feature-libs/customer-ticketing/components/customer-ticketing-reopen/customer-ticketing-reopen.component.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { ElementRef, ViewContainerRef } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { I18nTestingModule } from '@spartacus/core';
+import { LaunchDialogService, LAUNCH_CALLER } from '@spartacus/storefront';
+import { of } from 'rxjs';
+
+import { CustomerTicketingReopenComponent } from './customer-ticketing-reopen.component';
+
+describe('CustomerTicketingReopenComponent', () => {
+  let component: CustomerTicketingReopenComponent;
+  let fixture: ComponentFixture<CustomerTicketingReopenComponent>;
+
+  class MockLaunchDialogService implements Partial<LaunchDialogService> {
+    openDialog(
+      _caller: LAUNCH_CALLER,
+      _openElement?: ElementRef,
+      _vcr?: ViewContainerRef
+    ) {
+      return of();
+    }
+  }
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      imports: [I18nTestingModule],
+      declarations: [CustomerTicketingReopenComponent],
+      providers: [
+        { provide: LaunchDialogService, useClass: MockLaunchDialogService },
+      ],
+    }).compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(CustomerTicketingReopenComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
```

Comment:
Test for button not rendered when ticket status is not CLOSED
Maybe test open dialog
And test close dialog with X in top right, button cancel and submit